PageEdit/PageEdit.js

```diff
@@ -0,0 +1,36 @@
+const LinkToPortfolioPage = "../Portfolio/Portfolio.html";
+
+function redirectToPortfolioPage() {
+  window.location.href = LinkToPortfolioPage;
+}
+
+  const addExperienceButton = document.getElementById("addExperienceButton");
+  const experienceForm = document.getElementById("experienceForm");
+  const saveExperienceButton = document.getElementById("saveExperienceButton");
+  const experienceEntries = document.getElementById("experienceEntries");
+
+  addExperienceButton.addEventListener("click", () => {
+    experienceForm.style.display = "flex";
+  });
+
+  saveExperienceButton.addEventListener("click", () => {
+    const companyName = document.getElementById("companyName").value;
+    const startDate = document.getElementById("startDate").value;
+    const endDate = document.getElementById("endDate").value;
+    const description = document.getElementById("description").value;
+
+    const experienceEntry = document.createElement("div");
+    experienceEntry.classList.add("experience-entry");
+    experienceEntry.style.margin = "10px 0px";
```

Comment:
why adding a css class to the element and then also setting css styles explicitly below that?